src/cpu-info.spec.ts

```diff
@@ -1,20 +1,18 @@
-import { getCpuInfo } from './cpu-info';
+import { CpuInfo } from './cpu-info';
 
-describe('getCpuInfo', () => {
-  it('should get correct CPU information when available', () => {
+describe('CpuInfo', () => {
+  it('should get CPU information when available', () => {
```

Comment:
Should we center these tests around "number of logical cores" instead of "CPU information" now that we have a formal class that has separate methods?